compiler/luci/service/src/Nodes/CircleRange.cpp

```diff
@@ -84,6 +84,13 @@ loco::TensorShape Algorithm::visit(const luci::CircleRange *node)
   if (delta == 0)
     INTERNAL_EXN("Delta can not be zero");
 
+  /*
+   * **Pre-condition**
+   *    'limit - start' and 'delta' have the same sign.
+   *       c1. '(limit - start) >= 0' -> 'delta > 0'
+   *       c2. '(limit - start) < 0' -> 'delta < 0'
+   * https://github.com/tensorflow/tensorflow/blob/da82fa9d392d7c3cea3d86b516e55441e963b784/tensorflow/lite/kernels/range.cc#L49-L50
```

Comment:
```suggestion
   * https://github.com/tensorflow/tensorflow/blob/da82fa9/tensorflow/lite/kernels/range.cc#L49-L50
```
How about this ? 😀
hash values with 7 digits can be recognized.